src/app/pages/courses-page/actions-section/actions-section.component.html

```diff
@@ -1 +1,14 @@
-<p>actions-section works!</p>
+<div class="actions-section-container">
```

Comment:
I would not place all courses stuff within `pages` folder. It would be better to create a `CoursesModule`.

Reply:
Ok, will create separate module for courses stuff